### DISCUSSION TITLE
#50 Use Hand Sanitizer and border-radius

### DISCUSSION THREAD

**Discussion by rafehimself** [cssbattle/forum#143] [Bugs reports]
there is clearly something wrong with border-radius, that happened in #44 as well
![image](https://user-images.githubusercontent.com/58576132/137990312-071f40e6-7326-48f7-8486-c5b22b18ffc0.png)
```

<dt><p><c><c><d><d><d o><d i>
  <style>
  *{background:#1A4341;border-radius:20px;position:fixed}
  dt {
    border-radius:20px 20px 20px 20px;
    padding:70 50;
    margin:102 142;
    background:linear-gradient(#F3AC3C 40%,#998235 0) ;
    overflow:hidden;
    position:relative
  }
  p{background:#F3AC3C;padding:25;border-radius:55q;position:absolute;top:14;
  left:50;color:998235;box-shadow:-50px 0,-65px 28px,-10px 60px 0 40px}
    c{background:#998235;padding:10;top:100;left:280}c>c{top:130}
    d{background:#F3AC3C;padding:10 75;top:50;left:150}
    d>d{padding:20 10;top:50;left:280}[o]{left:190;top:60}
    [i]{padding:10 25;top:90;left:175;border-radius:10px 10px 0 0}
    
  </style>
```

I didn't even try to golf the code yet

**Comment by gunnarbittersmann** [comment]
While looking at smaller curves the checker is blind as a bat, it tends to be persnickety when it comes to bigger curves.

The problem is at the ~ shaped border between yellow and green. I don’t know if it’s the left or right side or the center. But I found that slightly tweaking one value fixes it. This is still a 💯 match: 

<img width="400" alt="Bildschirmfoto 2021-10-20 um 00 08 40" src="https://user-images.githubusercontent.com/409844/137998610-7bd80e90-9715-4829-8947-f8066cb8c8c0.png">

Where we see a difference, the checker sees none, and vice versa.

And for the record: I don’t cry for improvements in the checking algorithm; I see it as a part of the game. So might you.

**Reply by gunnarbittersmann** [reply]
See also https://github.com/cssbattle/forum/discussions/12#discussioncomment-452535

**Reply by rafehimself** [reply]
I'm not crying, just reporting bugs I found as I crawl through the targets 1 by 1, still 10 left to go 😝 
I guess that's what this subforum is for, whether the admins decide to do something with it, it's totally up to them,
but that particular problem is recurring on those values colors

**Reply by gunnarbittersmann** [reply]
The question is: Are these bugs in the checker, or in your solutions? I tend to say the latter. So, instead of reporting them, fix them! 😝
